membership-common/src/test/scala/com/gu/memsub/subsv2/reads/ChargeListReadsTest.scala

```diff
@@ -96,56 +100,56 @@ class ChargeListReadsTest extends AnyFlatSpec {
 
     val expected: ValidationNel[String, Benefit] = Success(Supporter)
 
-    Diff.assertEquals(expected, result)
+    result shouldMatchTo expected
```

Comment:
not sure if we were using the Diff.assertEquals method to help with type inference?  In the process of working it all out I ended up not using it any more, but I put explicit types on everything.  I'm not sure that's a bad thing really.